mdl/v1/block.go

```diff
@@ -0,0 +1,32 @@
+package mdl
+
+import (
+	"time"
+)
+
+type Block struct {
+	ID               string       `json:"id"`
+	Irreversible     bool         `json:"irreversible"`
+	Header           *BlockHeader `json:"header"`
+	TransactionCount int          `json:"transaction_count"`
+}
+
+type BlockHeader struct {
```

Comment:
We have the one in `eos-go`.. please use that one.. if there's anything different, then create a wrapper with a different name. Hopefully we don't need to do that.